lib/wallaby/query.ex

```diff
@@ -98,24 +98,26 @@ defmodule Wallaby.Query do
           | :option
           | :select
           | :file_field
+          | :attribute
   @type attribute_key_value_pair :: {String.t(), String.t()}
   @type selector ::
           String.t()
-          | :attribute_key_value_pair
+          | attribute_key_value_pair()
   @type html_validation ::
           :bad_label
           | :button_type
           | nil
   @type conditions :: [
-          count: non_neg_integer,
-          text: String.t(),
+          count: integer,
+          minimum: integer,
+          maximum: integer,
```

Comment:
Is there a case where we'd expect count, minimum, or maximum to be negative?

Reply:
Oh you're probably right, these should probably stay as `non_neg_integer`. I was changing the `at` to be correct because I believe underneath it will be passed to `Enum.at`, and I unnecessarily changed these three. I'll update!